packages/casimir/modules/nft-items/lib/constants/contentTypes.js

```diff
@@ -33,5 +33,5 @@ const contentTypesMapExtended = Object.keys(contentTypesMap).reduce((obj, key) =
   return o;
 }, {});
 
-export const projectContentTypes = [...Object.values(contentTypesMapExtended)]
+export const NftItemTypes = [...Object.values(contentTypesMapExtended)]
```

Comment:
```suggestion
export const nftItemTypes = [...Object.values(contentTypesMapExtended)]

```
в данном случаи с маленькой буквы. завтра вопрос еще задокументирую, но это не компонент и не класс - поэтому с маленькой буквы (как и было до изменений)

Reply:
это удалится с переходом на атрибуты